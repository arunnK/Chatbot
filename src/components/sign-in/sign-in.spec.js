import { create, act } from 'react-test-renderer';
import React from 'react';
import SignIn from './index';

let root;

test('sign in page', () => {
  act(() => {
    root = create(<SignIn />);
  });
  // make assertions on root
  expect(root.toJSON()).toMatchSnapshot();
});
