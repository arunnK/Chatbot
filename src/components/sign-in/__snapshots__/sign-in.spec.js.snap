// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sign in page 1`] = `
<div
  className="sign-in"
>
  <div
    className="sign-in-container"
  >
    <h3
      className="title"
    >
      Sign in with your email and password
    </h3>
    <h5
      className="error-text"
    />
    <form
      onSubmit={[Function]}
    >
      <div
        className="group"
      >
        <input
          className="form-input"
          label="Email"
          name="email"
          onChange={[Function]}
          required={true}
          type="email"
          value=""
        />
        <label
          className=" form-input-label"
        >
          Email
        </label>
      </div>
      <div
        className="group"
      >
        <input
          className="form-input"
          label="Password"
          name="password"
          onChange={[Function]}
          required={true}
          type="password"
          value=""
        />
        <label
          className=" form-input-label"
        >
          Password
        </label>
      </div>
      <div
        className="buttons"
      >
        <button
          type="submit"
        >
          Sign in
        </button>
      </div>
    </form>
  </div>
</div>
`;
